### PR TITLE
[CM T3] removed mentions of VERBOSE and HAVE_STDIO_H

### LINKED ISSUE #2313: [CM T0] remove VERBOSE
The macro VERBOSE and defined(HAVE_STDIO_H) does not make sense with the new logger anymore. These parts should be removed. (If there are printf inside, they should be ported to the new logger.)

### PR DESCRIPTION
**Pull request by mandoway**
## Basics

- [x] Short descriptions of your changes are in the release notes
      (added as entry in `doc/news/_preparation_next_release.md` which
      contains `_(my name)_`)
      **Please always add something to the release notes.**
- [x] Details of what you changed are in commit messages
      (first line should have `module: short statement` syntax)
- [ ] References to issues, e.g. `close #X`, are in the commit messages.
- [ ] The buildservers are happy. If not, fix **in this order**:
  - [ ] add a line in `doc/news/_preparation_next_release.md`
  - [ ] reformat the code with `scripts/dev/reformat-all`
  - [ ] make all unit tests pass
  - [ ] fix all memleaks
- [x] The PR is rebased with current master.

## Checklist

- [ ] I added unit tests for my code
- [ ] I fully described what my PR does in the documentation
      (not in the PR description)
- [ ] I fixed all affected documentation
- [ ] I added code comments, logging, and assertions as appropriate (see [Coding Guidelines](https://master.libelektra.org/doc/CODING.md))
- [ ] I updated all meta data (e.g. README.md of plugins and [METADATA.ini](https://master.libelektra.org/doc/METADATA.ini))
- [ ] I mentioned [every code](/.reuse/dep5) not directly written by me in [reuse syntax](https://reuse.software/)

## Review

- [ ] Documentation is introductory, concise, good to read and describes everything what the PR does
- [ ] Examples are well chosen and understandable
- [ ] Code is conforming to [our Coding Guidelines](https://master.libelektra.org/doc/CODING.md)
- [ ] APIs are conforming to [our Design Guidelines](https://master.libelektra.org/doc/DESIGN.md)
- [ ] Code is consistent to [our Design Decisions](https://master.libelektra.org/doc/decisions)

## Labels

- [ ] Add the "work in progress" label if you do not want the PR to be reviewed yet.
- [ ] Add the "ready to merge" label **if the basics are fulfilled** and no further pushes are planned by you.

---
Fixes #2313

### CODE REVIEW

**Review by kodebach — APPROVED** [review]
Good work

File: src/plugins/curlget/curlget.c [hunk 1]
```diff
@@ -490,6 +490,7 @@ static FILE * fetchFile (Data * data, int fd)
 		curl_easy_cleanup (curl);
 		return NULL;
 	}
+// todo replace VERBOSE
 #if VERBOSE
 	curl_easy_setopt (curl, CURLOPT_VERBOSE, 1L);
 #endif
```
Comment (markus2330) [inline src/plugins/curlget/curlget.c:496]: Simply remove the whole feature, I don't think this is needed. (If it would be needed, a plugin configuration would be the better choice as it would not require recompiliation.)

```suggestion
```

File: src/plugins/curlget/curlget.c [hunk 2]
```diff
@@ -774,6 +775,7 @@ int elektraCurlgetSet (Plugin * handle ELEKTRA_UNUSED, KeySet * returned ELEKTRA
 		curl = curl_easy_init ();
 		if (curl)
 		{
+// todo replace VERBOSE
 #if VERBOSE
 			curl_easy_setopt (curl, CURLOPT_VERBOSE, 1L);
 #endif
```
Comment (markus2330) [inline src/plugins/curlget/curlget.c:781]:
```suggestion
```

**Review by markus2330 — CHANGES_REQUESTED** [review]
Great job, please rebase.

File: doc/news/_preparation_next_release.md [hunk 3]
```diff
@@ -193,6 +193,10 @@ you up-to-date with the multi-language support provided by Elektra.
 - Added withKDB() extension which wraps the try block _(@Gratla)_
 - Added forEachKeyName() extension which iterates like in keyNameIterator _(@mandoway)_
 
+### Python
+
+- Deleted occurrences of removed property key.fullname _(@mandoway)_
+
```
Comment (kodebach) [inline doc/news/_preparation_next_release.md:199]: That's clearly not part of this PR

**Review by kodebach — CHANGES_REQUESTED** [review]

**Comment by mandoway** [comment]
Failed check is because of a broken link

**Comment by kodebach** [comment]
Please rebase, to fix the failing CI (the broken link is already whitelisted on master) and the merge conflicts.
